### ISSUE TITLE
DeprecationWarning: numpy boolean negative is deprecated

### ISSUE THREAD

**Issue by cancan101** [dnouri/nolearn#115]
```
/home/ubuntu/git/nolearn/nolearn/lasagne/util.py:150: DeprecationWarning: numpy boolean negative (the unary `-` operator) is deprecated, use the bitwise_xor (the `^` operator) or the logical_xor function instead.
```
from:
https://github.com/dnouri/nolearn/blob/62017dfb6566a3f89b16e9ddab06400026520320/nolearn/lasagne/util.py#L150

**Comment by dnouri** [comment]
ping @BenjaminBossan

**Comment by BenjaminBossan** [comment]
hmm, the warning message is confusing, *np.negative* and *np.logigical_xor* do different things. Should it not be *np.logical_not* instead?

**Comment by dnouri** [comment]
Closed with #126
